apps/web/src/pages/get-started/components/OpenWorkflowButton.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from '@emotion/styled';
+
+import { colors } from '@novu/design-system';
+import { INotificationTemplate } from '@novu/shared';
+
+import { ROUTES } from '../../../constants/routes.enum';
+import { useCreateTemplateFromBlueprint } from '../../../api/hooks';
+import { errorMessage } from '../../../utils/notifications';
+import { getBlueprintTemplateById, getTemplateById } from '../../../api/notification-templates';
+import { getWorkflowBlueprintDetails } from '../../../utils';
+import { TemplateCreationSourceEnum } from '../../templates/shared';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+import { OnboardingNodeEnum } from '../consts/types';
+
+export function OpenWorkflowButton({
+  blueprintIdentifier,
+  children,
+  node,
+}: {
+  blueprintIdentifier: string;
+  children: React.ReactNode;
+  node?: OnboardingNodeEnum;
+}) {
+  const segment = useSegment();
+
+  const { createTemplateFromBlueprint } = useCreateTemplateFromBlueprint({
+    onSuccess: (template) => {
+      const url = buildUrl(template, node);
+      openNewTab(url);
+    },
+    onError: (error) => {
+      errorMessage('Something went wrong while creating a template from the blueprint. Please try again later.');
+    },
+  });
+
+  const openNewTab = (url) => window.open(url, '_blank', 'noreferrer noopener');
+
+  const handleOpenWorkflowClick = async () => {
+    segment.track('[Get Started] Click Create Notification Template', {
+      templateIdentifier: blueprintIdentifier,
+      location: TemplateCreationSourceEnum.ONBOARDING_GET_STARTED,
+    });
```

Comment:
This way we will know if the user wanted to `Create` new use case workflow, or `Configure` use case node.

```suggestion
if(node){
    segment.track('Click Configure Use-Case Node [Get Started] ', {
      templateIdentifier: blueprintIdentifier,
      node: node
    });
    } else {
        segment.track('Click Create Workflow [Get Started] ', {
      templateIdentifier: blueprintIdentifier,
      node: node
    });
```

Reply:
Not sure I understand why we'd switch the `[ ]` to the end.

Also, this seems like something that could be handled with a ternary:

```suggestion
    const action = node ? 'Click Create Notification Template' : 'Click Create Workflow'; 
    segment.track(`[Get Started] ${action}`, {
      templateIdentifier: blueprintIdentifier,
      location: TemplateCreationSourceEnum.ONBOARDING_GET_STARTED,
    });
```

Reply:
Let's centralize the topic regarding the `[area]` in one place, i already addressed this question here :) 
https://github.com/novuhq/novu/pull/5212#discussion_r1496120200

Regarding the ternary, it has two cons in my opinion
* one event we have a `node` and another does not.
* as I love less code with reusable variables, in those scenarios where we log data it will be harder to debug code and search when the log is dynamic.

let me know what you think on the matter